### ISSUE TITLE
Quiz with multiple pages requiring password on every page

### ISSUE THREAD

**Issue by Alex-Jordan** [openwebwork/webwork2#2453]
In the 2.19 branch, I created a proctored test. In case it matters, these are the settings. (I haven't figured out if they matter.)

Open Date	6/26/23, 2:59 AM	
Close Date	7/3/24, 2:58 AM
Answers Available Date	7/3/24, 2:58 AM
Visible to Students	Yes
Restrict Access by Location	No
Hide Hints from Students	No
Assignment Type	Proctored Test

Test Time Limit	60
Cap Test Time at Close Date	Yes
Graded Submissions per Version	1
Time Interval for New Versions	0
Versions per Interval	1
Order Problems Randomly	No
Problems per Page	1
Show Scores on Finished Versions	Only after set answer date
Show Problems on Finished Versions	Only after set answer date

Proctor Authorization Type	Only start
Password	password

The quiz has 4 questions, so there are 4 pages. I start the test, and of course I must enter the password. But then if I go to another page, I am prompted again for the password. This seems like something is wrong.

This was reported in production by an instructor, but I reproduced it as above in the actual 2.19 branch.

**Comment by somiaj** [comment]
I thought #2435 fixed this? This was merged earlier this week, so an upgrade should hopefully fix things.

**Comment by Alex-Jordan** [comment]
@somiaj 

Well, that would make sense. I hadn't connected this with that. However I just reproduced the issue with a quiz that was on an up to date WW 2.19 branch at commit 54efd73228ec8f24ef68ef2a073cca4e6535fe93. So maybe it did not fix the issue. I will try again and confirm here.

**Comment by somiaj** [comment]
Maybe the fix wasn't complete enough and you have hit another case that causes a similar problem? When I tested the PR, I would need a proctor auth each and every page on a proctored quiz, and once I applied the patch, it worked fine and no longer required proctor auth for each page. I just tested again, and I am able to move between pages on a proctored quiz without needing proctor authorization for each page on a updated 2.19. So I am not able to reproduce this anymore on an updated 2.19. Assuming you updated all the js in `htdocs` with `npm ci` (just double checking)?
